server/src/main/java/org/opensearch/action/termvectors/MultiTermVectorsRequest.java

```diff
@@ -63,6 +63,8 @@ public class MultiTermVectorsRequest extends ActionRequest
         RealtimeRequest {
 
     String preference;
+
+    boolean realtime = true;
```

Comment:
Not sure if we should introduce a variable for a setting that is not set by users. In this case, we will be setting it to true everytime which we need to investigate if it's needed.

Reply:
Yes you are right. I also realized that MultiTermVectorsRequest itself cannot be a realtime request. So, there will be no point for us to route these to primary. So I have completely avoided routing MultiTermVectorsRequest to primary